doc/user/content/cli/_index.md

```diff
@@ -103,6 +104,23 @@ wish to configure `materialized` to only listen to, e.g. localhost connections,
 you can set `--listen-addr` to `localhost:6875`. You can also use this to change
 the port that Materialize listens on from the default `6875`.
 
+### Compaction window
+
+The `--logical-compaction-window` option specifies the size of the window behind
```

Comment:
how about dropping "the size of"? Ie, changing it to:
> specifies the window behind the current time for which

is perhaps less precise, but it reads much better.  We make it pretty clear that it's a window size in the "value of the option" section below

Reply:
Ack, agreed it was awkwardly worded. I rewrote it, and I think it's better, but probably still not perfect. Feel free to go to town once this lands! I think @frankmcsherry might take a pass too.